hs_core/models.py

```diff
@@ -2200,6 +2208,33 @@ def check_irods_files(self, stop_on_error=False, log_errors=True,
                 if stop_on_error:
                     raise ValidationError(msg)
 
+            if irods_public is not None:
+                irods_public = irods_public.lower() == 'true'
+
+            if irods_public is None or irods_public != django_public:
+                ecount += 1
+                if not django_public:  # and irods_public
+                    msg = "check_irods_files: resource {} public in irods, private in Django"\
+                        .format(self.short_id)
+                    if sync_ispublic:
+                        self.setAVU('isPublic', 'false')
```

Comment:
@alvacouch I see you moved this whole block for checking isPublic AVU sync out of try...except block, for more granular and accurate error message reporting, however, there are several places in this block ```self.setAVU()``` is called which could potentially raise iRODS exception as well although it is unlikely at this point since the resource collection is already there and irods connection should be established and working. But just for robustness, perhaps it is still better to catch the potential irods exceptions although they are unlikely to happen?